app/lib/task/backend.dart

```diff
@@ -329,26 +331,30 @@ class TaskBackend {
     String packageName, {
     bool updateDependants = false,
   }) async {
-    var lastVersionCreated = initialTimestamp;
-    await withRetryTransaction(_db, (tx) async {
-      final pkgKey = _db.emptyKey.append(Package, id: packageName);
+    final package = await packageBackend.lookupPackage(packageName);
```

Comment:
If we're willing to use a cached or potentially old value, then we can just take it as a parameter, instead of `packageName`.

I didn't do that because I wanted to reload the `Package` entity if the transaction has a conflict.